packages/wasm-dpp/lib/test/fixtures/getDashPayContractFixture.js

```diff
@@ -0,0 +1,16 @@
+const dashPaySchema = require('@dashevo/dashpay-contract/schema/dashpay.schema.json');
```

Comment:
I'm not sure if I get why we need all these fixtures implemented in JS . Shouldn't we just use the ones from rust dpp?

Reply:
Yep, absolutely! It's just the old test, and I'm replacing things one by one. This PR is already pretty big, so I would love to merge this one first before we continue with other stuff